lib/browser/api/browser-window.js

```diff
@@ -17,6 +17,15 @@ BrowserWindow.prototype._init = function () {
   // Create WebContentsView.
   this.setContentView(new WebContentsView(this.webContents))
 
+  const nativeSetBounds = this.setBounds
+  this.setBounds = (bounds, animate) => {
```

Comment:
Can you please pass all arguments after the first one as an array?

```js
this.setBounds = (bounds, ...rest) => {
  <...>
  nativeSetBounds.call(this, bounds, ...rest)
}
```

Should be less error-prone.